user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Grid.md

```diff
@@ -0,0 +1,69 @@
+---
+uid: DashboardGrid
+---
+
+# Grid
+
+> [!IMPORTANT]
+> At present, this component is only available in preview, if the [ReportsAndDashboardsDynamicVisuals](xref:Overview_of_Soft_Launch_Options#reportsanddashboardsdynamicvisuals) soft-launch option is enabled. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
```

Comment:
The component is available without softlaunch from 10.4.1. Between 10.2.12 and 10.4.1 it's available with the softlaunch. This note makes me think I can only use the component with the flag enabled from 10.4.1 onwards.